sequenceSearch/views.py

```diff
@@ -43,6 +48,9 @@ def search(request):
             alignments = cache.get((input, evalue, maxHits))
             if not alignments:
                 alignments = sequencetools.blast(query=input, evalue=evalue, max_target_seqs=maxHits)
+                if alignments == 0:
```

Comment:
Let's use a try/except here when calling blast, instead of doing an in stream signal of zero. For the except, we need to catch only the desired exception, and not all exceptions, e.g. by doing `except <exception type>:` instead of `except:`.